core/services/install-services.sh

```diff
@@ -25,7 +25,7 @@ cd /home/pi/services/ardupilot_manager/ && python3 setup.py install
 cd /home/pi/services/helper/ && python3 setup.py install
 
 # Version Chooser service:
-cd /home/pi/services/versionchooser/ && python3 setup.py install
+cd /home/pi/services/versionchooser/ && pip install .
```

Comment:
Should we move everything to pip install for consistency ?
Also, there is no comment explaining the reason why pip is used here and not in the following lines

Reply:
I think we can change them all to pip. Any thoughts @rafaellehmkuhl ?

Reply:
Definitely agree. Was going to do this on the commonwealth PR but forgot.